src/ConfigProvider.php

```diff
@@ -22,8 +22,8 @@ public function __invoke() : array
     public function getAuthenticationConfig() : array
     {
         return [
-            'username' => '', // provide a custom field name for the username
-            'password' => '', // provide a custom field name for the password
+            'username' => 'username', // provide a custom field name for the username
+            'password' => 'password', // provide a custom field name for the password
```

Comment:
Maybe it's better to set them `null` by default?

Reply:
That could be better yes.
My thought process, however, is that as a user who is not aware of what options are available and their defaults so I could structure my application accordingly, ConfigProvider is one of first stops at checking what the package needs so I can either override it or, in this case, to find out what my PSR7 request needs to contain to complete my login request. 

If you do not consider that as a valid point, I will change it 👍